setup.py

```diff
@@ -1,3 +1,25 @@
-import setuptools
+from setuptools import setup, find_packages
 
-setuptools.setup(setup_requires=['pbr'], pbr=True)
+
+def readme():
+    with open('README.md') as f:
+        return f.read()
+
+
+setup(
+    name='nmstate',
+    version='0.0.1',
+    description='Declarative network manager API',
+    author="Red Hat",
```

Comment:
Not sure if a company can be the author. It should probably be Edy's, yours and my name.

Reply:
I just copy the string from `setup.cfg`

Reply:
ok, then we can keep this

Reply:
or better change it later

Reply:
I have change it to Edward Haas as he is the auther of the fist git commit.

We can record our names in AUTHORS file later, that file could also be useful when we need re-license or other all-buy-in changes.